app/src/main/java/org/tokend/template/features/redeem/accept/view/ScanRedemptionActivity.kt

```diff
@@ -88,6 +88,11 @@ class ScanRedemptionActivity : BaseActivity() {
     override fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?) {
         super.onActivityResult(requestCode, resultCode, data)
 
+        if(requestCode == ACCEPT_REQUEST_CODE) {
+            tryOpenQrScanner()
+            return
+        }
+
         if (resultCode == Activity.RESULT_CANCELED) {
             finish()
```

Comment:
Reopen scanner no matter what the result was

Reply:
I did. This part is to catch scanner closing.

Reply:
Sorry